src/components/network-chart/NetworkChartRow.js

```diff
@@ -6,6 +6,7 @@
 import React from 'react';
```

Comment:
```suggestion
import * as React from 'react';
```

This should fix the Flow error around React.Fragment.